tf2_eigen/CMakeLists.txt

```diff
@@ -14,8 +14,9 @@ find_package(geometry_msgs REQUIRED)
 find_package(tf2 REQUIRED)
 find_package(tf2_ros REQUIRED)
 
+find_package(eigen3_cmake_module REQUIRED)
 find_package(Eigen3 REQUIRED)
-message(STATUS "Using Eigen3 include dirs: ${EIGEN3_INCLUDE_DIR}")
+message(STATUS "Using Eigen3 include dirs: ${Eigen3_INCLUDE_DIRS}")
```

Comment:
I know you didn't add it, but since you are touching it, I think we should just remove this line.  While it is informational, you could say the same about literally *any* dependency that we use `find_package` on, so I'm not sure why Eigen is special.

Reply:
Removed in 8dfffa1